debian/control

```diff
@@ -9,9 +9,15 @@ Build-Depends: debhelper (>= 9),
                texlive-full,
                inkscape,
                cython,
+               cython3,
                python-numpy,
-               python-all-dev (>= 2.7.1)
+               python-all-dev (>= 2.7.1),
+               python3-setuptools,
+               python3-sphinx (>= 1.1.3),
+               python3-numpy,
+               python3-all-dev (>= 2.7.1)
```

Comment:
That's really a small version number for Python 3. I'd suggest to use 3.5 as a minimum, as that's the oldest one we use for testing (on Ubuntu 16.04).